contracts/_mocks/LZEndpointMock.sol

```diff
@@ -0,0 +1,339 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.11;
+pragma abicoder v2;
```

Comment:
Is it the abicode necessary after 0.8.11?

Reply:
Prolly not, just copy pasta'd from LZ repo and probably changed the ver and didn't remove it. Will remove 🖖 N1